collections/README.md

```diff
@@ -32,115 +28,116 @@ The following Link relations shall exist in the Landing Page (root).
+`child` relations are useful for supporting browsing a STAC API as if it were a static catalog.
 
-The following Link relations should exist in the `/collections` endpoint response.
+The following Link relations shall exist in the `/collections` endpoint response.
 
 | **rel** | **href**       | **From**  | **Description** |
 | ------- | -------------- | --------- | --------------- |
 | `root`  | `/`            | STAC Core | The root URI    |
 | `self`  | `/collections` | OAFeat    | Self reference  |
 
-The following Link relations should exist in the `/collections/{collectionId}` endpoint response.
+The following Link relations shall exist in the Collection object returned from the `/collections/{collectionId}` endpoint.
 
 | **rel**  | **href**                            | **From**  | **Description**                            |
 | -------- | ----------------------------------- | --------- | ------------------------------------------ |
 | `root`   | `/`                                 | STAC Core | The root URI                               |
 | `parent` | `/`                                 | OAFeat    | Parent reference, usually the root Catalog |
 | `self`   | `/collections/{collectionId}`       | OAFeat    | Self reference                             |
-| `items`  | `/collections/{collectionId}/items` | OAFeat    | Items within Collection                    |
+
+Additionally, these relations may exist for the `/collections/{collectionId}` endpoint:
+
+| **rel**  | **href**                            | **From**  | **Description**                            |
+| -------- | ----------------------------------- | --------- | ------------------------------------------ |
+| `canonical`        | various              | STAC Core      | The child STAC Catalogs & Collections. Provides curated paths to get to STAC Collection and Item objects |
+
+Usually, the `self` link in a Collection must link to the same URL that was used to request
+it. However, implementations may choose to have the canonical location of the Collection be
+elsewhere. If this is done, it is recommended to include a `rel` of `canonical` to that location.
 
 ## Endpoints
 
-| Endpoint                      | Returns    | Description                                                          |
-| ----------------------------- | ---------- | -------------------------------------------------------------------- |
-| `/collections`                | JSON       | Object with a list of Collections contained in the catalog and links |
-| `/collections/{collectionId}` | Collection | Returns single Collection JSON                                       |
+| Endpoint | Returns | Description |
+| -------- | ------- | ----------- |
+| `/`                           | Catalog        | Landing Page and root Catalog |
+| `/api` (recommended)          | OAFeat OpenAPI | The OpenAPI service description |
```

Comment:
This is a bit ambiguous. Is the whole row recommended or just the name of the endpoint? (I guess the latter?)

Reply:
just the name of the endpoint. I've clarified the text now by adding it to the description.